File: week01_1.py
room = input()
list(room)
number = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
for i in range(0,len(room)):
    for j in range(0, 10):
        if(j == int(room[i])):
            number[j] = number[j] + 1
number[6] = number[6] + number[9]
if number[6] % 2 == 1:
    number[6] = number[6] + 1
number[6] = number[6] / 2
temp = 0
for i in range(0, 9):
    if temp < int(number[i]):
        temp = number[i]
print(int(temp))

File: baekjoon/01주차/1475-방-번호/우진/week01_1.java
import java.util.Scanner;

public class week01_1 {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String room = sc.nextLine();
        String[] number = room.split("");
        int[] num = new int[9];
        for(int i = 0; i < number.length; i++)
        {
            if(number[i].equals("0"))
                num[0]++;
            else if(number[i].equals("1"))
                num[1]++;
            else if(number[i].equals("2"))
                num[2]++;
            else if(number[i].equals("3"))
                num[3]++;
            else if(number[i].equals("4"))
                num[4]++;
            else if(number[i].equals("5"))
                num[5]++;
            else if(number[i].equals("6") || number[i].equals("9"))
                num[6]++;
            else if(number[i].equals("7"))
                num[7]++;
            else if(number[i].equals("8"))
                num[8]++;          
        }
        if(num[6] % 2 == 1)
            num[6]++;
        num[6] = num[6] / 2;
        int temp = 0;
        for(int i = 0; i < num.length; i++)
        {
            if(temp < num[i])
                temp = num[i];
        }
        System.out.print(temp);
    }
}
